packages/data/src/api/tokenById/tokenById.types.ts

```diff
@@ -0,0 +1,52 @@
+export interface NftTokenData {
+  animationUrl: string | null;
```

Comment:
If it's not super painful, might be nice to standardize these fields as `camelCase`. Not sure if we added that one to the lint rules

Reply:
for that we would have to do a mapper , or change on the query itself , and change the fields coming from the backend